client/blocks/comments/post-comment-list.jsx

```diff
@@ -137,7 +137,7 @@ class PostCommentList extends React.Component {
 
 	renderCommentsList( commentIds ) {
 		return <ol className="comments__list is-root">
-			{ commentIds.reverse().map( ( commentId ) => this.renderComment( commentId ) ) }
+			{ map( commentIds, ( commentId ) => this.renderComment( commentId ) ) }
```

Comment:
This was experimentation with having everything on the selectors. Better not go that route now. Restored on 0176a2ecd2104228eeb7c24259844cdc11e51de7.